options/locale/locale_en-US.ini

```
@@ -1999,6 +1999,7 @@ settings.delete_notices_2 = - This operation will permanently delete the <strong
 settings.delete_notices_fork_1 = - Forks of this repository will become independent after deletion.
 settings.deletion_success = The repository has been deleted.
 settings.update_settings_success = The repository settings have been updated.
+settings.update_settings_no_unit = The repository should have at least one unit.
```

```suggestion
settings.update_settings_no_unit = The repository should allow at least some sort of interaction.
```
(Not sure about this translation.
However, we cannot speak about "unit"s for users.
They don't have a concept of what we mean by that…)